third_party/web_locale_keymap/lib/web_locale_keymap/locale_keymap.dart

```diff
@@ -41,6 +41,9 @@ class LocaleKeymap {
       return eventKeyCode;
     }
     if (result == null) {
+      if ((eventCode ?? '').isEmpty && (eventKey ?? '').isEmpty) {
```

Comment:
nit: could this be `eventCode == null && eventKey == null` ?

Reply:
No. We're passing empty strings there during some tests unfortunately.